packages/synthetic-chain/src/lib/vat-status.js

```diff
@@ -62,6 +62,14 @@ const makeSwingstore = db => {
       if (!targetVat) throw Error(`vat not found: ${vatName}`);
       return targetVat;
     },
+    /** @param {string} vatName */
```

Comment:
this one's a substring, not a name

would it help to support regexp?

Reply:
I changed the name.

I can't think of a reason to want a regexp here. YAGNI